shpc/main/container/docker.py

```diff
@@ -33,13 +33,19 @@ def __init__(self):
             )
         super(DockerContainer, self).__init__()
 
+    def get_shell_path(self):
```

Comment:
Perhaps shell_path as a property?
```suggestion
    @property
    def shell_path(self):
```